yelp_package/dockerfiles/itest/chronos/Dockerfile

```diff
@@ -14,20 +14,20 @@
 
 FROM ubuntu:trusty
 
-RUN apt-get update && apt-get -y install apt-transport-https
+RUN DEBIAN_FRONTEND=noninteractive apt-get update > /dev/null && DEBIAN_FRONTEND=noninteractive apt-get -y install apt-transport-https > /dev/null
```

Comment:
Just do

    ENV DEBIAN_FRONTEND=noninteractive

So you don't have to repeat yourself 8 times in the dockerfile